### DISCUSSION TITLE
Missed import SegAutoMaskGenerator in segment-anything-video/metaseg/webapp/app.py

### DISCUSSION THREAD

**Discussion by apatynko** [kadirnar/segment-anything-video#45] [Q&A] [answered]
Hi,

I was playing with your example of webapp and found that the code cannot import
`from metaseg import SegAutoMaskGenerator`
Can you suggest what is meant under 'SegAutoMaskGenerator' ?

**Comment by kadirnar** [comment, marked as answer]
> Hi,
> 
> I was playing with your example of webapp and found that the code cannot import `from metaseg import SegAutoMaskGenerator` Can you suggest what is meant under 'SegAutoMaskGenerator' ?

Did you look at the readme file?

**Reply by apatynko** [reply]
yes. Examples from readme file work. I just was not able to find `SegAutoMaskGenerator` anywhere. If it is missed then forget. For my internal test, I found a workaround. So it is not important for me now.
